packages/forms/src/components/BasicForm.tsx

```diff
@@ -108,14 +108,18 @@ function BasicForm<TFieldValues extends FieldValues>({
         /**
          * Iterates through all cached values touching and dirtying the fields
          * so we can display any errors and unsaved changes
+         *
+         * Note: There is a lot of type hacking with `as` here since
+         * `react-hook-form` does not export the required types
```

Comment:
Worth opening an issue with the library?

✍️ https://github.com/react-hook-form/react-hook-form/issues/new?assignees=&labels=feature+request%2C+waiting-up-vote&template=feature_request.md&title=

Reply:
I'm not sure, there is no real reason to need that type. We are just doing something very funky to render those cached values that doesn't really make sense for the library. We could try though, who knows 🤷‍♀️